src/frontend/Sider/Journalføring/Felles/utils.ts

```diff
@@ -0,0 +1,75 @@
+import { ISelectOption, MultiValue, PropsValue, SingleValue } from '@navikt/familie-form-elements';
+
+import { LogiskVedlegg } from '../../../typer/dokument';
+
+export type MultiSelectValue = { label: string; value: string };
+
+export const dokumentTitler: string[] = [
+    'Anke på tilbakekreving',
+    'Arbeidsforhold',
+    'Avtale / Avgjørelse om samvær',
+    'Bekreftelse fra barnevernet',
+    'Bekreftelse på termindato',
+    'Bekreftelse på tilsynsutgifter',
+    'Bekreftelse på utdanning / utgifter',
+    'Endring i sivilstand',
+    'Enslig mor eller far som er arbeidssøker',
+    'Eklæring om samlivsbrudd',
+    'EØS dokument',
+    'Forespørsel',
+    'Fødselsmelding/Fødselsattest',
+    'Grunnblankett',
+    'Inntektsopplysninger',
+    'Klage på tilbakekreving',
+    'Klage/Anke',
+    'Krav om gjenopptak av ankesak',
+    'Medisinsk dokumentasjon',
+    'Merknader i ankesak',
+    'Oppholdstillatelse',
+    'Refusjonskrav/faktura',
+    'Rettsavgjørelse',
+    'Skatteopplysninger',
+    'Stevning',
+    'Søknad om overgangsstønad',
+    'Søknad om skolepenger',
+    'Søknad om barnetilsyn',
+    'Tilmelding til NAV som reell arbeidssøker ved krav om overgangsstønad',
```

Comment:
Litt usikker på hvilke vedlegg vi burde ha her, men burde kanskje ha med `Søknad om støtte til pass av barn` iaf. Det kan ev. gjøres som egen oppgave

```suggestion
    'Arbeidsforhold',
    'Avtale / Avgjørelse om samvær',
    'Bekreftelse fra barnevernet',
    'Bekreftelse på tilsynsutgifter',
    'Bekreftelse på utdanning / utgifter',
    'EØS dokument',
    'Forespørsel',
    'Grunnblankett',
    'Inntektsopplysninger',
    'Klage på tilbakekreving',
    'Klage/Anke',
    'Krav om gjenopptak av ankesak',
    'Medisinsk dokumentasjon',
    'Merknader i ankesak',
    'Oppholdstillatelse',
    'Refusjonskrav/faktura',
    'Rettsavgjørelse',
    'Skatteopplysninger',
    'Stevning',
    'Søknad om støtte til pass av barn',
```

Reply:
Kan godt kjøre grundig tilbakemelding nå, så blir det mindre å gå gjennom senere

Reply:
https://favro.com/organization/98c34fb974ce445eac854de0/4d617346d79341c7fbd9a40a?card=NAV-20201

Reply:
La til `Søknad om støtte til pass av barn`. Har en liste med oversikt over oppfølgingsoppgaver her: https://favro.com/organization/98c34fb974ce445eac854de0/4d617346d79341c7fbd9a40a?card=NAV-19955